dyld[17136]: Library not loaded: @loader_path/../Python
  Referenced from: <3F2D0F69-E25A-3340-8B52-46400654C50E> /Users/chanon_mike/Library/Application Support/pypoetry/venv/bin/python3.11
  Reason: tried: '/Users/chanon_mike/Library/Application Support/pypoetry/venv/bin/../Python' (no such file), '/usr/local/lib/Python' (no such file), '/usr/lib/Python' (no such file, not in dyld cache)

Traceback:

  File "<stdin>", line 923, in main
  File "<stdin>", line 562, in run
